opennms-base-assembly/src/main/filtered/etc/org.opennms.features.eif-adapter.cfg

```diff
@@ -0,0 +1,4 @@
+# Interface on which to listen
```

Comment:
We can omit including this in the base assembly and rely on the Blueprint's defaults instead.